server/handlers.go

```diff
@@ -45,3 +45,9 @@ func (server HTTPServer) serveAPISpecFile(writer http.ResponseWriter, request *h
 
 	http.ServeFile(writer, request, absPath)
 }
+
+// listOfGroups returns the list of defined groups
+func (server *HTTPServer) listOfGroups(writer http.ResponseWriter, request *http.Request) {
```

Comment:
where is `server.Groups` set? I only saw `config.Groups`

Reply:
In `New` function, inside `server/server.go` :-)

Reply:
@JoseLSegura I only see `server.Config` there

Reply:
@JoseLSegura ah, nevermind, im just stupid